man/org.freedesktop.systemd1.xml

```diff
@@ -1291,7 +1289,13 @@ node /org/freedesktop/systemd1 {
       and terminate all units that aren't dependencies of it. If <literal>ignore-dependencies</literal>, it
       will start a unit but ignore all its dependencies. If <literal>ignore-requirements</literal>, it will
       start a unit but only ignore the requirement dependencies. It is not recommended to make use of the
-      latter two options. On completion, this method returns the newly created job object.</para>
+      latter two options. On completion, if successful, this method returns the newly created job object
+      which has been enqueued for asynchronouos activation. Callers that want to track the outcome of the
```

Comment:
asynchronouos ← typo

Reply:
I'll never, ever get that right on the first try